lib/service/storage/elasticsearch.js

```diff
@@ -2121,7 +2121,9 @@ class ElasticSearch extends Service {
           _index: alias
         }
       });
-      esRequest.body.push(extractedDocuments[i]._source);
+      if (_source) {
```

Comment:
A unit test should be added/modified for this

Reply:
I added functionnal tests on this part of the code

Reply:
I know but on Kuzzle core we need to write unit tests as well to ensure maximal quality and maintainability

Reply:
I talked with Yannick about the unit tests, and we think it's unnecessary because the ES tests use mock function.